server/utils/jwt.go

```diff
@@ -0,0 +1,41 @@
+package utils
+
+import (
+	"net/http"
+
+	"github.com/codeday-labs/bookworms/server/db"
+	"github.com/golang-jwt/jwt"
+)
+
+type Claims struct {
+	User db.User
+	jwt.StandardClaims
+}
+
+var Jwt_key = []byte("kdksksm2039")
```

Comment:
It is not a good practice to hard code jwt secret in your source, I have created an environment variable in Vercel called `JWT_SECRET` with the same value instead.